gst/nnstreamer/registerer/nnstreamer.c

```diff
@@ -102,9 +104,11 @@ gst_nnstreamer_init (GstPlugin * plugin)
   NNSTREAMER_INIT (plugin, transform, TRANSFORM);
   NNSTREAMER_INIT (plugin, if, IF);
   NNSTREAMER_INIT (plugin, rate, RATE);
+#if (ENABLE_NNS_QUERY == 1)
```

Comment:
How about defining a macro something like ```NNSTREAMER_QUERY_INIT``` which is valid only if ```ENABLE_NNS_QUERY == 1```?

Reply:
Same above.